hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/api/protocolrecords/package-info.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
this isn't required, it isn't a javadoc AFAIK, is it also giving failures

Reply:
I rolled back the code, but found that it cannot be compiled by java-doc, we still need to add * to line 1 of package-info.java.

Reply:
Ok

Reply:
There are bunch of package-info.java file which doesn't do this. like:
https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/tools/package-info.java
https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/http/package-info.java
Why it is a problem here only

Reply:
Thank you for your help reviewing the code!

These package-info.java will have this problem in JDK11, we can see the following test report
https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-5151/8/artifact/out/branch-javadoc-hadoop-common-project_hadoop-common-jdkUbuntu-11.0.17+8-post-Ubuntu-1ubuntu220.04.txt

I found that only when there is a new javadoc warning or error, the Java doc error reminder will be triggered. From my personal point of view, we should fix it, so that this module can be completely compiled through JDK11.